pkg/ingester/flush.go

```diff
@@ -246,7 +246,10 @@ func (i *Ingester) shouldFlushChunk(chunk *chunkDesc) bool {
 	}
 
 	if time.Since(chunk.lastUpdated) > i.cfg.MaxChunkIdle {
-		chunk.closed = true
```

Comment:
This was removed from `shouldFlushChunk` for a few reasons:
1) It's only previous reference, `collectChunksToFlush `, was immediately followed by duplicate closing code: https://github.com/grafana/loki/blob/2179ed1018133b54ad7f018fff5f57e9699b9655/pkg/ingester/flush.go#L230-L232
2) It's now used by `sweepStream` to support deduplicating logic and this new location should not alter the `chunk.closed` field
3) `shouldFlushChunk` doesn't sound like it should mutate the underlying `*chunkDesc`